docs/MS Excel/Формат ячеек.md

```diff
@@ -0,0 +1,39 @@
+≡ Краткое руководство Markdown
+
+# Формат ячеек
+
+**Общий формат**
+
+Общий формат — это числовой формат по умолчанию, Excel применяется при его введите. В большинстве случае числа, отформатированные в формате Общий, отображаются так, как вы их введите. Однако если ширины ячейки недостаточно для демонстрации всего числа, то формат Общий округляет числа, которые имеют десятичность.
+
+**Числовой формат**
+
+С помощью числовых форматов можно изменять внешний вид чисел, в том числе даты и времени, не меняя при этом фактические числа. Числовой формат не влияет на значение в ячейке, которое Excel использует для вычислений. Фактическое значение отображается в строке формул.
+
+**Текстовый формат**
+
+Текстовый формат позволяет определить выравнивание, шрифт и размер текста в ячейке, а также добавить заливку и рамку. Форматирование дат и времени позволяет выбрать различные способы отображения даты, времени и их комбинаций, а также добавить формулы для математических операций со временем.
+
+**Дата. Что такое дата для Exel. Как вводить даты.**
+
+Дата в Excel — это порядковый номер дня при начале счета с 1 января 1900 года.
+Функция ДАТА возвращает последовательный серийный номер, соответствующий определенной дате
+Вставка фиксированного значения даты или времени в ячейку Excel
+1.	Выделите на листе ячейку, в которую требуется вставить текущую дату или время.
+
+2.	Выполните одно из действий, указанных ниже.
+
+    •	Чтобы вставить текущую дату, нажмите клавиши CTRL+; (точка с запятой).
+
+    •	Чтобы вставить текущее время, нажмите клавиши CTRL+SHIFT+; (точка с запятой).
+
+    •	Чтобы вставить текущие дату и время, нажмите клавиши CTRL+; (точка с запятой), затем клавишу ПРОБЕЛ, а после этого — CTRL+SHIFT+; (точка с запятой).
```

Comment:
[ch-md-error-ul-style](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-ul-style)